docs/versioning.md

```diff
@@ -0,0 +1,112 @@
+
+The following sections list our guidelines for incrementing `AssemblyVersion` and `AssemblyFileVersion`.
+
+### AssemblyVersion
+
+`AssemblyVersion` is used by the runtime to locate the correct version of the assembly to load. `AssemblyVersion` has the following format for CsWinRT generated assemblies:
+
+| | AssemblyVersion Format |
+|-|-|
+| **WinRT.Runtime.dll** | *\<major>.\<minor>.0.0* **\*** |
+| **Microsoft.Windows.SDK.NET.dll** |  *10.0.<windows_build>.<projection_update>* |
+
+**\*** **Note**: this format for the runtime assembly takes effect starting with `AssemblyVersion` *1.1.0.0*, which aligns with the first .NET5 servicing release.
+
+The *<windows_build>* refers to the Windows SDK build number. As of .NET5 RTM, the supported *<windows_build>* versions are 17763, 18362, and 19041.
+
+The guidelines for incrementing `AssemblyVersion` are as follows. Note that `AssemblyVersion` increments to either assembly are independent of one another.
+
+1. For *WinRT.Runtime.dll*:
+    * Do *not* increment: if we only have backwards compatible bug-fixes.
+        * **Note**: this differs from Semantic Versioning rules
+    * Increment *\<minor>* or *\<major>*: if there are any changes to the public API surface.
+2. For *Microsoft.Windows.SDK.NET.dll*:
+    * Increment *\<projection_update>*: if there are changes to the CsWinRT generated API surface for the Windows SDK projections, for example if we expose more functions or if we discover we are not projecting a function that we should be.
+    * Update *\<windows_build>*: if there is a new version of the Windows API surface, which would occur with a new Windows OS release and corresponding SDK. This will involve listing a new SDK projection package for the new Windows version.
+
+### AssemblyFileVersion
+
+`AssemblyFileVersion` is used for deployment and can mark assemblies with the same AssemblyVersion. It can be viewed in Windows from **File Properties** -> **File Version**. `AssemblyFileVersion` has the following format for CsWinRT generated assemblies:
+
+| | AssemblyFileVersion Format | 
+|-|-|
+| **WinRT.Runtime.dll** | *\<major>.\<minor>.\<patch>.\<build>* |
+| **Microsoft.Windows.SDK.NET.dll** |  *10.0.<windows_build>.<package_update>* |
+
+`AssemblyFileVersion` will be incremented with each release, regardless of whether there are API changes. This means if `AssemblyVersion` does not increment, we will still increment `AssemblyFileVersion`. The guidelines for incrementing `AssemblyFileVersion` are as follows.
+
+1. For *WinRT.Runtime.dll*:
+    * Increment *\<minor>* or *\<major>*: if there is an increment to `AssemblyVersion`. This update would be in correspondence with the update to `AssemblyVersion`.
+    * Increment *\<patch>* and/or *\<build>*: if there is no increment to `AssemblyVersion`.
```

Comment:
we have yet to set patch to nonzero (but could, to be semver compliant)
build is always automatically incremented